modules/mobile/app/models/mobile/v0/adapters/va_facilities.rb

```diff
@@ -31,6 +31,15 @@ def map_appointments_to_facilities(appointments, facilities)
 
         private
 
+        def map_facilities_id(facilities)
+          facilities&.map do |facility|
+            next if facility.id.nil?
```

Comment:
FYI, this will leave nils in the resulting array. You may want to compact the result.

Reply:
compacting the result caused errors. it looks like the home appts won't have facilities ids (so the nils are needed)